### PR TITLE
Add functionality

### PR DESCRIPTION
**Pull request by ViachaslauKabak**
* Add labels
* Add webhooks
* Create group
* Create project
* Create master user

### CODE REVIEW

File: tasks/create_master_user.yml [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+  set_fact:
+    gitlab_master_id: "{{ item.id }}"
+  loop: "{{ user_list_output.json }}"
+  when:
+    - user_exists
+    - item.username == gitlab_master_username
+
+- name: "User debug"
+  debug:
+    msg: "User id is {{ gitlab_master_id }}"
```
Comment (vutkin) [inline tasks/create_master_user.yml:48]: We still need it?

File: tasks/create_master_user.yml [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+# - name: "Get token"
+#   set_fact:
+#     gitlab_master_token: "{{ item.token }}"
+#   loop: "{{ token_list_output.json }}"
+#   when:
+#     - token_exists
+#     - item.name == gitlab_master_token_name
+
+- name: "Token debug"
```
Comment (vutkin) [inline tasks/create_master_user.yml:131]: We still need it?

File: tasks/create_project.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+- name: "Get project ID"
+  set_fact:
+    gitlab_project_id: "{{ item.id }}"
+  loop: "{{ projects_list_output.json }}"
+  when:
+    - project_exists
+    - item.path_with_namespace == gitlab_project_group + '/' + gitlab_project_name
+
+- name: "Project debug"
```
Comment (vutkin) [inline tasks/create_project.yml:44]: We still need it?

Comment (ViachaslauKabak) [reply tasks/create_project.yml:44]: yes - we need it to get the ID of created user, group and project for further usage

File: tasks/create_master_user.yml (hunk 2 above)
Comment (ViachaslauKabak) [reply tasks/create_master_user.yml:131]: yes - we need it to get the ID of created user, group and project for further usage

File: tasks/create_master_user.yml (hunk 1 above)
Comment (ViachaslauKabak) [reply tasks/create_master_user.yml:48]: yes - we need it to get the ID of created user, group and project for further usage

**Pull request merged**